public/js/src/articleView.js

```diff
@@ -0,0 +1,25 @@
+(function(exports) {
+  function ArticleView(article = new Article) {
+    this.article = article
+  }
+
+  ArticleView.prototype.returnHTMLstring = function() {
+    let stringHTML;
+    if (this.article.url.length !== 0) {
+      fetch('http://news-summary-api.herokuapp.com/guardian?apiRequestUrl=http' + this.article.url.substring(5) + '?show-fields=body')
+        .then(response => response.json())
+        .then(data => {
+          let result = data.response.content;
+          let content = result.fields.body
+          stringHTML = content;
+          document.getElementById('app').innerHTML = stringHTML;
+        })
+    } else {
+      stringHTML = '<ul><li><div>' + this.article.headline + '</div></li></ul>'
+      document.getElementById('app').innerHTML = stringHTML;
```

Comment:
The part of your code that attaches new element to the DOM should be in the controller not the view (to be consistent with what you do for article list). Also because this function is called `returnHTMLstring` so it should just return the string, not also change the DOM.